src/platform/forms/address/helpers.js

```diff
@@ -129,6 +129,7 @@ export function formatAddress(address) {
   let street = address.addressOne || '';
   if (address.addressOne && address.addressTwo) street += ', ';
   if (address.addressTwo) street += address.addressTwo;
+  if (address.addressTwo && address.addressThree) street += ',';
```

Comment:
This'll work. But we could take this opportunity to replace this code with something a little more elegant (IMO):

```js
street = [address.addressOne, address.addressTwo, address.addressThree]
  .filter((e) => e)
  .join(', ');
```

Reply:
Great suggestion! Updated 👍

Reply:
cool. I really only mentioned this because we had a conversation about this in a similar situation a couple weeks ago https://github.com/department-of-veterans-affairs/vets-website/pull/12111#discussion_r403280455